drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -196,6 +203,7 @@ function DragNDropTemplates(url_name) {
                         'data-zone_id': zone.id,
                         'data-zone_align': zone.align,
                         'role': 'button',
+                        'aria-describedby': getItemPropertyFromSet(items_in_zone_descriptions, "displayName").join(", "),
```

Comment:
Agreed, could be better worded. Is the user exposed to the term "zone" at all? Otherwise it might be a bit clunky. Maybe just "Items in here:" would be better.